### PR TITLE
Fix crashes

### PR DESCRIPTION
**Pull request by adrianschroeter**
visible in delayed jobs

### CODE REVIEW

**Comment by adrianschroeter** [comment]
needs also a backport to 2.8 most likely (not verified)

File: src/api/lib/workers/accept_requests.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def initialize
 
   def perform
     User.current = User.find_by_login('Admin')
-    BsRequest.find_requests_to_accept.each do |r|
+    BsRequest.to_accept.each do |r|
```
Comment (bgeuken) [inline src/api/lib/workers/accept_requests.rb]: That was done in August 2013 by 7a92356c95bc7e91a105bde0017900c2c522f00a

How did we manage to not run into this earlier?^^

**Comment by bgeuken** [comment]
LGTM

**Comment by adrianschroeter** [comment]
On Donnerstag, 23. März 2017, 04:04:28 CET wrote Björn Geuken:
> bgeuken commented on this pull request.
> 
> 
> 
> > @@ -4,7 +4,7 @@ def initialize
>  
>    def perform
>      User.current = User.find_by_login('Admin')
> -    BsRequest.find_requests_to_accept.each do |r|
> +    BsRequest.to_accept.each do |r|
> 
> That was done in August 2013 by 7a92356c95bc7e91a105bde0017900c2c522f00a
> 
> How did we manage to not run into this earlier?^^

I wonder as well ... short answer is we have no test case.  But I know that 
automated request accepting still worked some month ago ...

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg) 
 
Maxfeldstraße 5                         
90409 Nürnberg 
Germany

**Comment by bgeuken** [comment]
> adrianschroeter commented 11 minutes ago 
> I wonder as well ... short answer is we have no test case. But I know that automated request accepting still worked some month ago ...

Calling perform would have failed for sure.

**Pull request merged**